src/modules/commander/failsafe/framework.cpp

```diff
@@ -287,10 +291,11 @@ void FailsafeBase::notifyUser(uint8_t user_intended_mode, Action action, Action
 					events::ID("commander_failsafe_enter"),
 				{events::Log::Critical, events::LogInternal::Warning},
 				"{3}: switching to {2}", mavlink_mode, failsafe_action, failsafe_cause);
+
+				mavlink_log_critical(&_mavlink_log_pub, "Failsafe activated\t");
 			}
 		}
 
```

Comment:
How about
```suggestion
if (action != Action::Warn) {
	mavlink_log_critical(&_mavlink_log_pub, "Failsafe activated\t");
}
```
To avoid duplication which on top could lead to additional flash use.